dockstore-webservice/src/main/java/io/dockstore/webservice/core/Version.java

```diff
@@ -390,6 +396,14 @@ public void setImages(Set<Image> images) {
         this.images = images;
     }
 
+    public Set<ImageReference> getImageReferences() {
```

Comment:
This `.dockstore.yml` reference seems to be one string, the `ImageReference` seems to be one string as well (i.e. missing architecture from images) so why is this a set?

Reply:
Coming back after looking at the rest, if anything this seems more like a `path` as in `primaryDescriptorPath` or `readMePath` 

If we do go down the route of having a `ImageReference` which is separate from `Image` which I'm not completely on board with, this `ImageReference` should just be a string like the other paths to avoid the temptation of storing more things in the `image_reference` table